### PR TITLE
[Flink AI Flow] Skip events from default namespace in workflow_event_processor

### PR DESCRIPTION
**Pull request by bgeng777**
## What is the purpose of the change

Skip events from default namespace in workflow_event_processor

## Brief change log

  - Skip events from `default` namespace in workflow_event_processor. Because there are no context extractor under this name. This keyword is preserved for external events like MODEL_GENERATED. Such events should not be handled by workflow_event_processor.

## Verifying this change

This change is already covered by existing tests, such as *(please describe tests)*.

### CODE REVIEW

File: flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py [hunk 1]
```diff
@@ -67,17 +68,21 @@ def run(self) -> None:
...
-        subscribed_workflow = self._get_subscribed_workflow(event, project_name)
-
-        for workflow in subscribed_workflow:
+        subscribed_workflow = []
+        projects = self.store.list_project(sys.maxsize, 0)
+        for project_name in projects:
+            workflows = self._get_subscribed_workflow(event, project_name)
+            if len(workflows) > 0:
+                for workflow in workflows:
+                    subscribed_workflow.append((project_name, workflow))
```
Comment (Sxnan) [inline flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py:77]: We can use the `extend` method to make the code a little bit more consice.

File: flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py [hunk 2]
```diff
@@ -67,17 +68,21 @@ def run(self) -> None:
...
+
+        for workflow_tuple in subscribed_workflow:
+            project_name, workflow = workflow_tuple
             self._handle_event_for_workflow(project_name, workflow, event)
 
     def _get_subscribed_workflow(self, event, project_name):
         workflows: List[WorkflowMeta] = self.store.list_workflows(project_name=project_name)
+        print(workflows)
```
Comment (Sxnan) [inline flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py:85]: `print` is not recommanded. Maybe use debug level logging.

**Review by Sxnan — COMMENTED** [review]
The patch looks good overall. I have left two minor comments. Thanks for the patch.

File: flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py (hunk 2 above)
Comment (bgeng777) [reply flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py:85]: removed

File: flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py (hunk 1 above)
Comment (bgeng777) [reply flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py:77]: `extend` is different from `append` as `extend` only accepts `list` arg. 
Here `subscribed_workflow` is a list of tuple and the codes aim to add a new tuple into the list. `extend` can also work if we write codes like: `subscribed_workflow.extend([(project_name, workflow)])`. But it seems somehow not clean enough.

Comment (Sxnan) [reply flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py:77]: I overlook the fact that it is a tuple we are adding to the list. We can keep it.

**Review by Sxnan — APPROVED** [review]
LGTM

**Pull request merged**